index.html

```diff
@@ -32,11 +32,12 @@ <h4 id="challenge-text" class="challenge-text">press any key to start the game</
       </div>
     </div>
           <div class = "row">
-          <div class="instructions col s9 offset-s6">
+          <div class="instructions col s3 offset-s8">
               <p>Directions:</p>
-              <p>Keep the bird afloat, by typing the correct answer.</p>
+              <p>Keep the bird alive by typing the correct answer.</p>
```

Comment:
maybe we should say- ``by typing the text on the screen`` or something like that.